plugins/slack/alerta_slack.py

```diff
@@ -198,6 +198,10 @@ def post_receive(self, alert, **kwargs):
             LOG.debug("Alert severity %s is included in SLACK_SEVERITY_FILTER list, thus it will not be forwarded to Slack." % alert.severity)
             return
 
+        if alert.severity in ['ok', 'normal', 'cleared'] and alert.previous_severity in SLACK_SEVERITY_FILTER:
```

Comment:
Someone might have set a custom normal severity so you need to do something like this...
```
if alert.severity in ['ok', 'normal', 'cleared', app.config.get('DEFAULT_NORMAL_SEVERITY')] and alert.previous_severity in SLACK_SEVERITY_FILTER:
```